### PR TITLE
add enable_apis submodule for enabling redis api

### PR DESCRIPTION
**Pull request by milesmatthias**
Followed suggestion to use CFT service module, but it doesn't appear to be working:

```
       Error: Invalid for_each set argument

         on .terraform/modules/memstore.enable_apis.project-services/terraform-google-modules-terraform-google-project-factory-a1584fe/modules/project_services/main.tf line 21, in resource "google_project_service" "project_services":
         21:   for_each                   = toset([for api in var.activate_apis : api if var.enable_apis])

       The given "for_each" argument value is unsuitable: "for_each" supports maps
       and sets of strings, but you have provided a set containing type dynamic.
```

### CODE REVIEW

**Comment by morgante** [comment]
@milesmatthias Can you try providing a PR on the services module to fix this?

**Comment by milesmatthias** [comment]
@morgante Sure, happy to. Just acknowledging it's increased scope.

**Comment by milesmatthias** [comment]
waiting for a new version release of https://github.com/terraform-google-modules/terraform-google-project-factory.

File: enable_apis/main.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/**
```
Comment (morgante) [inline enable_apis/main.tf]: @milesmatthias Please move this into `modules/enable_apis`

Comment (milesmatthias) [reply enable_apis/main.tf]: done

**Comment by milesmatthias** [comment]
Can this be merged?

File: variables.tf [hunk 2]
```diff
@@ -25,6 +25,11 @@ variable "project" {
   type        = string
 }
 
+variable "enable_apis" {
+  description = "Flag for enabling redis.googleapis.com in your project"
+  default     = "false"
```
Comment (morgante) [inline variables.tf]: We shouldn't change the type or default.

File: variables.tf [hunk 3]
```diff
@@ -83,9 +87,3 @@ variable "labels" {
   type        = map(string)
   default     = null
 }
-
-variable "enable_apis" {
-  description = "Enable required APIs for Cloud Memorystore."
-  type        = bool
```
Comment (morgante) [inline variables.tf]: Don't remove the type or default.

Comment (morgante) [inline variables.tf]: Don't remove the type or default.

Comment (milesmatthias) [reply variables.tf]: Okay, I included the type & default and matched it with the project services module: https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/modules/project_services/variables.tf#L21-L24

**Pull request merged**